alsa_settings/README.md

```diff
@@ -0,0 +1,79 @@
+# ALSA settings
+
+SOF test case execution requires certain DSP and mixer configurations being set
+on the DUT (Device Under Test). An important part of this configuration is the
+signal levels adjusted to the hardware setup which runs the tests, including its
+codec and loopback devices.
+
+The `./alsa_settings/` directory contains custom ALSA configurations which
+are currently applied at the SOF CI pipeline's hardware platforms.
+When the SOF tests run in other environment, these confugurations can serve
+as a reference.
+
+These configurations include ALSA settings which are considered essential
+for the SOF tests and expected at the appropriate DUT by default. They allow
+ALSA settings tweaking in along with SOF and test developmnent, if needed.
+
+Please note that in some cases the settings are tuned to CI DUT 'harnesses':
+external codecs with their versions, audio cards at loopback connections, etc.
+In the future, most of these settings should be moved to the DUT's default config
+out of the test case scope, so only the test-specific settings will remain here.
```

Comment:
There should be a simple way to identify which setting is part of which category in each file using using some "standardized" comment convention and/or sections.